master/buildbot/changes/base.py

```diff
@@ -55,24 +55,59 @@ def _unclaimService(self):
 class ReconfigurablePollingChangeSource(ChangeSource):
     pollInterval = None
     pollAtLaunch = None
-
-    def checkConfig(self, name=None, pollInterval=60 * 10, pollAtLaunch=False):
+    pollRandomDelayMin = None
+    pollRandomDelayMax = None
+
+    def checkConfig(
```

Comment:
Can we keep aligning the arguments with the opening parentheses like in the rest of the codebase?

Reply:
Sorry about that, I come from black so I didn't think twice about this. Moved back to inline while staying under the 100 mark.